cli/azd/pkg/project/service_target_dotnet_containerapp.go

```diff
@@ -393,6 +400,17 @@ func (at *dotnetContainerAppTarget) Deploy(
 		if err != nil {
 			return nil, fmt.Errorf("deploying bicep template: %w", err)
 		}
+		if slices.ContainsFunc(deploymentResult.Resources,
+			func(resource *armresources.ResourceReference) bool {
+				webAppType := string(azapi.AzureResourceTypeWebSite)
+				if foundWebApp := strings.Contains(*resource.ID, webAppType); foundWebApp {
+					resourceName = strings.Split(*resource.ID, webAppType+"/")[1]
```

Comment:
I'd recommend using `arm.ParseResourceID(resourceId)` method instead of the manual parsing here.